Mathlib/CategoryTheory/Bicategory/NaturalTransformation/Oplax.lean

```diff
@@ -32,57 +35,42 @@ universe w₁ w₂ v₁ v₂ u₁ u₂
 
 variable {B : Type u₁} [Bicategory.{w₁, v₁} B] {C : Type u₂} [Bicategory.{w₂, v₂} C]
 
-/-- If `η` is an oplax natural transformation between `F` and `G`, we have a 1-morphism
+/-- If `η` is an oplax transformation between `F` and `G`, we have a 1-morphism
 `η.app a : F.obj a ⟶ G.obj a` for each object `a : B`. We also have a 2-morphism
 `η.naturality f : F.map f ≫ app b ⟶ app a ≫ G.map f` for each 1-morphism `f : a ⟶ b`.
 These 2-morphisms satisfies the naturality condition, and preserve the identities and
 the compositions modulo some adjustments of domains and codomains of 2-morphisms.
 -/
-structure OplaxNatTrans (F G : OplaxFunctor B C) where
+structure OplaxTrans (F G : OplaxFunctor B C) where
```

Comment:
Why not just `Oplax.Trans`?
```suggestion
structure Trans (F G : OplaxFunctor B C) where
```

Reply:
Because there should be three transformations between oplax functors (oplax transformations, lax transformations and strong transformations) if I have understood correctly

Reply:
So `Oplax.OplaxTrans` should indicate that this is an oplax transformation between oplax functors (so later I can also have `Lax.OplaxTrans`.

Reply:
Ok, so we are going to have about 9 times of transformations between lax/oplax/pseudofunctors?!

Reply:
In theory yes, but I just want to have `Oplax.OplaxTrans` and `Pseudofunctor.StrongTrans` for now. I do not know in what situation one needs these other transformations, but in case someone wants to define them, I want to set the library up so that this is not too painful (perhaps if someone wishes to do that, some sort of automation could be a good idea).